build.sbt

```diff
@@ -15,7 +19,10 @@ lazy val commonSettings = Seq(
   libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value,
   libraryDependencies += "com.github.scopt" %% "scopt" % "3.7.0",
   libraryDependencies += "org.scala-lang.modules" % "scala-jline" % "2.12.1",
+  libraryDependencies += "com.typesafe.play" %% "play-json" % "2.6.10",
   addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
+  unmanagedBase := (chipyardRoot / unmanagedBase).value,
+  allDependencies := allDependencies.value.filterNot(_.organization == "edu.berkeley.cs"),
```

Comment:
Does this work? I think some things (is it barstools?) use "berkeley". This is fragile but it's hard to think of a better alternative.

Reply:
The robust way would be to just create a blacklist of libraries (org %% name) and filter on those. We can punt on that though.